storage/postgres/migrations/20200722413800_alter_broker_platform_credentials_active.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
usually these come with a corresponding `.down.sql` script rollback the migration if needed, can you add it?

Reply:
done